crates/host/src/wasmbus/config.rs

```diff
@@ -50,6 +50,12 @@ pub struct Host {
     pub provider_shutdown_delay: Option<std::time::Duration>,
     /// Configuration for downloading artifacts from OCI registries
     pub oci_opts: OciConfig,
+    /// Whether to allow loading actor or provider components from the filesystem
+    pub allow_file_load: bool,
+    /// Whether or not structured logging is enabled
+    pub enable_structured_logging: bool,
+    /// Log level to pass to capability providers to use. Should be parsed from a [`tracing::Level`]
+    pub log_level: String,
```

Comment:
How about an enum?

Reply:
I wanted to use `tracing::Level` and then just `.to_string()` it but it didn't meet the wasmbus_rpc Serialize trait requirements 🤔